packages/now-node/src/index.ts

```diff
@@ -290,8 +290,13 @@ export async function build({
   config = {},
   meta = {},
 }: BuildOptions) {
-  const shouldAddHelpers = config.helpers !== false;
-  const awsLambdaHandler = config.awsLambdaHandler as string;
+  const shouldAddHelpers = config.helpers
+    ? config.helpers !== false
+    : process.env.NODEJS_HELPERS !== '0';
+
+  const awsLambdaHandler = config.awsLambdaHandler
+    ? config.awsLambdaHandler as string
+    : process.env.NODEJS_AWS_HANDLER_NAME;
```

Comment:
This might hold a value such as `handler`, whereas `awsLambdaHandler` contained `index.handler`. In turn, we need to split at `.` and take the second part of the string.